src/sentry/static/sentry/app/views/stream/searchBar.jsx

```diff
@@ -473,7 +473,7 @@ const SearchBar = createReactClass({
     if (this.props.disabled) rootClassNames.push('disabled');
 
     return (
-      <div className={classNames(rootClassNames)}>
+      <div className={classNames(rootClassNames)} style={{zIndex: 1001}}>
```

Comment:
Can we move this zIndex into `utils/themes`

Reply:
It's already there, unfortunately the z-indexes uses for emotion styles follow a different system to the ones used in LESS files, this is a workaround until we migrate to styled components